pyproject.toml

```diff
@@ -49,7 +49,7 @@ django = ["django"]
 docs = ["sphinx", "sphinx-rtd-theme"]
 linting = ["flake8", "black", "isort", "mypy", "types-python-dateutil", "yamllint", "vulture"]
 sqlalchemy = ["sqlalchemy"]
```

Comment:
Haven't used this before, but looks sweet!
I might lock a specific version in the optional dependencies just to prevent breaking updates.

Reply:
Ah yes ``pytest-xdist`` does speed up the tests massively.